test/com/core/tictactoe/BoardTests.java

```diff
@@ -0,0 +1,171 @@
+package com.core.tictactoe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+
+public class BoardTests {
+
+    private Board board;
+    private Player player;
+    private Player player2;
+
+    @BeforeEach
+    void instantiateBoard() {
+        board = new Board();
+        player = new Player("X");
+        player2 = new Player("Y");
+    }
+
+    @Test
+    void BoardIsCreatedWithPlacesArray() {
```

Comment:
All tests start with a lower case even if it's a class name.

Reply:
Actually you are testing 'Board' so redundant to say it in the test name.
I always like the way rspec and other  language testing frameworks use 'it' so in my head for Java tests I still say: it 'is created with places array' for instance.

Reply:
Done.